### ISSUE TITLE
Crash while trying to import data to Elden Ring regulations file.

### ISSUE THREAD

**Issue by mikeydoom** [vawser/Yapped-Rune-Bear#35]
System Info: Windows 11
CPU i9 9900k
GPU: RTX 3080 Ti

_**Error:**_

```
?See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.
************** Exception Text **************
System.FormatException: String was not recognized as a valid Boolean.
   at System.Boolean.Parse(String value)
   at Yapped.FormMain.ImportParamData(ParamWrapper wrapper, Boolean isSilent) in E:\Program Files (x86)\SteamLibrary\steamapps\common\DARK SOULS III\Modding\Mods\Cinders\Tools\C# Projects\Yapped\Yapped\FormMain.cs:line 2226
   at Yapped.FormMain.ImportDataToolStripMenuItem_Click(Object sender, EventArgs e) in E:\Program Files (x86)\SteamLibrary\steamapps\common\DARK SOULS III\Modding\Mods\Cinders\Tools\C# Projects\Yapped\Yapped\FormMain.cs:line 515
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4470.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
Yapped
    Assembly Version: 1.1.2.0
    Win32 Version: 1.1.2
    CodeBase: file:///C:/Users/dbgtf/Downloads/Elden%20Ring%20Mods/Yapped-Rune-Bear-1_09/Yapped.exe
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4488.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4488.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4395.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4341.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4470.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4161.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
SoulsFormats
    Assembly Version: 1.1.8126.38820
    Win32 Version: 1.1
    CodeBase: file:///C:/Users/dbgtf/Downloads/Elden%20Ring%20Mods/Yapped-Rune-Bear-1_09/SoulsFormats.DLL
----------------------------------------
************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.
For example:
<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>
When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

I have the latest version of .net framework installed. So I'm not sure what the issue is.

**Comment by SmoofBrane** [comment]
I was having the exact same issue. I tried changing , (comma) values to . (period) values and vice versa, double checked the .csv itself, but it didn't work.
For reference: the mods I was using were the `regulation.bin` from 'Fun Overhaul' and the `Bullet.csv` from the 'Magic Overhaul'. The issue was that the .csv was malformed. What I did to properly import it was:

1. make a backup of the "clean" `regulation.bin` file (ALWAYS MAKE BACKUPS)
2. download the magic overhaul `regulation.bin` (NOT the .csv) and place in elden ring `Game` folder
4. mass export
5. delete all files EXCEPT the `Bullet.csv` (read `Yapped\res\GR\Data\Bullet.csv`) extracted from the magic overhaul `regulation.bin`
6. download the fun overhaul `regulation.bin`, place and OVERWRITE in `Game` folder
7. mass import (which only imports the remaining `Bullet.csv` into the fun overhaul `regulation.bin`)
8. success

I hope this helps people figure out how to perhaps extract .csv's from mods and merge them successfully with other mods as is the case in my situation. Good luck
